app/src/components/fields/fieldFilterEntity/fieldFilterEntity.jsx

```diff
@@ -13,15 +13,16 @@ export const FieldFilterEntity = ({
   stretchable,
   smallSize,
   onRemove,
+  vertical,
 }) => (
-  <div className={cx('field-filter-entity', { stretchable }, { small: smallSize })}>
-    <span className={cx('entity-name')}>{title}</span>
+  <div className={cx('field-filter-entity', { stretchable, small: smallSize, vertical })}>
+    <span className={cx('entity-name', { vertical: 'vertical' })}>{title}</span>
```

Comment:
`cx('entity-name', { vertical: 'vertical' })` looks like mistake. Usually properties in object in classnames should be `true` or `false`, but here it's string so vertical always will be true.

If vertical really always should be true you should write `cx('entity-name', 'vertical')`.
But if vertical should depends of `vertical` variable, you should write `cx('entity-name', { vertical })`.

Please look at the documentation of classnames library: https://github.com/JedWatson/classnames/blob/master/README.md